Integrations/integration-McAfeeNSM.yml

```diff
@@ -0,0 +1,679 @@
+    - contextPath: NSM.Alerts.Event
+      description: Alert Event
+    - contextPath: NSM.Alerts.Attack
+      description: Alert's Attack
+    - contextPath: NSM.Alerts.Attacker
+      description: The Attacker who committed the attack
+    - contextPath: NSM.Alerts.Target
+      description: The Attack's target
+    - contextPath: NSM.Alerts.MalwareFile
+      description: Malware File used in the attack
+    description: Get a single alert details
+  - name: nsm-get-ips-policies
+    arguments:
+    - name: domain_id
+      required: true
+      default: true
+      description: Domain ID
+    outputs:
+    - contextPath: NSM.IPSPolicies.ID
+      description: IPS Policy ID
+    - contextPath: NSM.IPSPolicies.Name
+      description: IPS Policy Name
+    - contextPath: NSM.IPSPolicies.DomainID
+      description: IPS Policy Domain ID
+    - contextPath: NSM.IPSPolicies.IsEditable
+      description: whether the IPS policy is editable
+    - contextPath: NSM.IPSPolicies.VisibleToChildren
+      description: whether the IPS Policy is visible to domain's children
+    description: Get IPS Policies in a domain
+  - name: nsm-get-ips-policy-details
+    arguments:
+    - name: policy_id
+      required: true
+      default: true
+      description: IPS Policy ID
+    outputs:
+    - contextPath: NSM.IPSPolicies.ID
+      description: IPS Policy ID
+    - contextPath: NSM.IPSPolicies.Name
+      description: IPS Policy Name
+    - contextPath: NSM.IPSPolicies.Description
+      description: IPS Policy information
+    - contextPath: NSM.IPSPolicies.CreatedTime
+      description: Policy creation time
+    - contextPath: NSM.IPSPolicies.IsEditable
+      description: whether the IPS policy is editable
+    - contextPath: NSM.IPSPolicies.VisibleToChildren
+      description: whether the IPS Policy is visible to domain's children
+    - contextPath: NSM.IPSPolicies.Version
+      description: IPS Policy Version
+    - contextPath: NSM.IPSPolicies.InboundRuleSet
+      description: Inbound Rule Set
+    - contextPath: NSM.IPSPolicies.OutboundRuleSet
+      description: Outbound Rule Set
+    - contextPath: NSM.IPSPolicies.ExploitAttacks
+      description: A list of exploit attacks related to the IPS Policy
+    description: Get the policy details for the specific IPS policy
+  - name: nsm-get-attacks
+    arguments:
+    - name: attack_id
+      default: true
+      description: Particular Attack ID
+    outputs:
+    - contextPath: NSM.Attacks.ID
+      description: Attack ID
+    - contextPath: NSM.Attacks.Name
+      description: Attack Name
+    - contextPath: NSM.Attacks.Severity
+      description: Attack Severity
+    - contextPath: NSM.Attacks.Direction
+      description: Attack Direction
+    - contextPath: NSM.Attacks.Category
+      description: Attack Category
+    description: Get all available attack definitions in NSM
+releaseNotes: Add new integration, supporting alerts querying and updating, IPS policies querying
```

Comment:
Not critical but `releaseNotes` are not necessary for new integrations/scripts/etc